src/style.css

```diff
@@ -0,0 +1,80 @@
+* {
+  box-sizing: border-box;
+  font-weight: 300;
+  color: #3d4e61;
+  font-family: "avenirFont", sans-serif;
+  font-size: 16px;
+}
+
```

Comment:
in `*` we can use only `box-sizing: border-box;` other styles are forbidden. Move it to the appropriate selector